test/parallel/test-os.js

```diff
@@ -130,12 +130,35 @@ switch (platform) {
       const actual = interfaces['Loopback Pseudo-Interface 1'].filter(filter);
       const expected = [{ address: '127.0.0.1', netmask: '255.0.0.0',
                           mac: '00:00:00:00:00:00', family: 'IPv4',
-                          internal: true }];
+                          internal: true, cidr: '127.0.0.1/8' }];
       assert.deepStrictEqual(actual, expected);
       break;
     }
 }
 
+// assert cidr prop present all interface info
+function flatten(arr) {
+  const _flatten = function(p, c) {
+    if (Array.isArray(c)) p = p.concat(c.reduce(_flatten, []));
+    else p.push(c);
+    return p;
+  };
+
+  return arr.reduce(_flatten, []);
+};
+const netmaskToCIDRSuffixMap = new Map(Object.entries({
+  '255.0.0.0': 8,
+  '255.255.255.0': 24,
+  'ffff:ffff:ffff:ffff::': 64,
+  'ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff': 128
```

Comment:
@silverwind So I moved the code out but requiring `internal/os` fails with a module not found. Do I need to register it somewhere? Also, where do tests for internal modules go? I see the`test` folder in `lib/internal` only has tests for `unicode`.